addons/ammunition/CfgMagazines.hpp

```diff
@@ -2,40 +2,40 @@ class CfgMagazines {
     class CA_Magazine;
 
     CLASS(8Rnd_P_000): CA_Magazine {
-        author = "TyroneMF";
+        author = $STR_tacgt_ammo_Author;
```

Comment:
```suggestion
        author = $STR_TACGT_Ammo_Author;
```
Format for stringtables.